Doc/whatsnew/3.10.rst

```diff
@@ -225,6 +225,238 @@ See :class:`typing.Callable`, :class:`typing.ParamSpec`,
 
 (Contributed by Ken Jin in :issue:`41559`.)
 
+PEP 634: Structural Pattern Matching
+------------------------------------
+
+Structural pattern matching has been added in the form of a *match statement*,
+a data object (sequences, mappings, primitive data types as well as class
+instances) and *case statements* of patterns with associated actions. Pattern
+matching enables programs to extract information from complex data types,
+branch on the structure of data, and apply specific actions based on different
+forms of data.
+
+Syntax and operations
+~~~~~~~~~~~~~~~~~~~~~
+
+The generic syntax of pattern matching is::
+
+    def function(subject):
+        match {subject}:
```

Comment:
Why the curly braces? (If this is "meta variables" as I suspect I'd think we usually use `<angular brackets>` over `{curlies}`. Plus, `subject` is the parameter of the function so maybe it shouldn't be a meta variable at all?)